recipes/st_tree/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from conans.errors import ConanInvalidConfiguration
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+class STTreeConan(ConanFile):
+    name = "st_tree"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "A fast and flexible c++ template class for tree data structures"
+    topics = ("stl", "container", "data-structures")
+    homepage = "https://github.com/erikerlandson/st_tree"
+    generators = "cmake"
```

Comment:
```suggestion
```

As you don't have a cmake wrapper which would consume conanbuildinfo.cmake, you don't need any generator. But it doesn't harm

Reply:
I agree with you.
I forget this recipe is header only.